devices/pine64-pinetab/README.adoc

```diff
@@ -0,0 +1,46 @@
+= PineTab
```

Comment:
```suggestion
= Pine64 PineTab
```

I know the PinePhone README doesn't start with Pine64, but that's a weirdness from the setup, and a documentation bug you found :).  The [produced page](https://mobile.nixos.org/devices/pine64-pinephone.html) does not actually use this title. This title *should* reflect the final title of the page.

Reply:
Fixed, and also added a separate commit to fix the PinePhone README.